welkin/models/calendar.py

```diff
@@ -137,14 +127,10 @@ def id(self, value=None):
 
         self.details[0]["workHoursId"] = value
 
-    def create(self, repeating=True):
-        self.startDateTime = datetime.now(tz=timezone.utc)
-        if repeating:
-            self.endDateTime = self.startDateTime.replace(
-                year=self.startDateTime.year + 50
-            )
-
-        return super().post(f"{self._client.instance}/calendar/work-hours")
+    def create(self, *args, **kwargs):
```

Comment:
for consistency with the rest of the wrapper you can leave out the *args, **kwargs here unless you have a specific reason for how one would use them, e.g. if you show it in a test